### DISCUSSION TITLE
How would I break up categories?

### DISCUSSION THREAD

**Discussion by wzrdk3lly** [jpanther/congo#39] [Q&A] [answered]
I'm trying to understand how the categories work? 

Do I create categories by:

Content/
---->blog
        ------->first-post.md
---->CTF Writeup
        -------->second-post.md

 Or do I need to specify "category: 'blog'" and "category: 'ctf writeup'" at the top of my md files?

**Comment by jpanther** [comment, marked as answer]
I guess this depends on what you are trying to achieve with the category. If you are using it as a taxonomy, then yes, you add `category: category_name` to the front matter.

If you are referring to having separate sections on your site, you can simply follow the normal Hugo content directory rules. Either name the files with the desired name, or you can use page bundles and create a sub-directory with an `index.md` file.

```
.
└── content
    └── about
    |   └── index.md        // <- https://.../about/
    ├── blog
    |   ├── firstpost.md    // <- https://.../blog/firstpost/
    |   └── secondpost
    |       └── index.md    // <- https://.../blog/secondpost/
    └── writing
        ├── first.md        // <- https://.../writing/first/
        └── second.md       // <- https://.../writing/second/
```

Check out the exampleSite to see how the content is organised into different sections. You can also read more about this in the Hugo [Content Organization](https://gohugo.io/content-management/organization/) docs.

**Comment by wzrdk3lly** [comment]
Thanks @jpanther !
